game/view/gameplay/actionhud/controlhint.lua

```diff
@@ -0,0 +1,70 @@
+
+local FONT    = require 'view.helpers.font'
+local RES     = require 'resources'
+local PROFILE = require 'infra.profile'
+local COLORS  = require 'domain.definitions.colors'
+
+local Class   = require "steaming.extra_libs.hump.class"
+local ELEMENT = require "steaming.classes.primitives.element"
+local vec2    = require 'cpml' .vec2
```

Comment:
single quote ou double quote? don't you guys have a standard?

Reply:
A much bigger problem than this PR alone. Opening #1458 to use luafmt as soon as possible.